### PR TITLE
books: 700__9 rule add, refactor errors logging

### LINKED ISSUE #182: Implement Rule for 700__9#BEARD#
Regarding: 700__9#BEARD#, we could add a "curated_relation" property to:
cds-dojson/cds_dojson/schemas/records/books/ymls/elements/id.yml
When #BEARD#, it curated_relation -> false. 
Is this the meaning of this information?

_Originally posted by @agentilb in https://github.com/CERNDocumentServer/cds-dojson/issues/179#issuecomment-456335412_

### LINKED ISSUE #185: Ignore fields 694__a and 694__9
ignore field 694__a and 694__9

### PR DESCRIPTION
**Pull request by kprzerwa**
* (closes #182)

* (closes #185)

### CODE REVIEW

File: cds_dojson/marc21/fields/utils.py [hunk 1]
```diff
@@ -363,11 +363,14 @@ def build_contributor_books(value):
...
 
+    value_9 = clean_val('9', value, str)
+
     contributor = {
         'ids': _extract_json_ids(value, 'schema') or None,
         'full_name': value.get('name') or clean_val('a', value, str),
         'email': clean_email(value.get('email')),
-        'role': _get_correct_books_contributor_role(value.get('e', 'author'))
+        'role': _get_correct_books_contributor_role(value.get('e', 'author')),
+        'curated_relation': value_9 if value_9 == '#BEARD#' else None
```
[the remark this answers is not in the document]
Comment (agentilb) [reply cds_dojson/marc21/fields/utils.py]: For Books and Proceedings, 700__9 is inly used for #BEARD#.

[the remark this answers is not in the document]
Comment (kprzerwa) [reply cds_dojson/marc21/fields/utils.py]: also I have checked the old records and #BEARD# appear only when the ID-> source is CDS, is it correct?

File: cds_dojson/marc21/fields/books/book.py [hunk 2]
```diff
@@ -282,6 +286,7 @@ def urls(self, key, value):
     try:
         clean_val('y', value, str, manual=True)
     except ManualMigrationRequired as e:
+        e.subfield = 't'
```
[the remark this answers is not in the document]
Comment (kprzerwa) [reply cds_dojson/marc21/fields/books/book.py]: true, thanks!

File: cds_dojson/marc21/fields/utils.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (agentilb) [reply cds_dojson/marc21/fields/utils.py]: I think so, since this info has been added by CDS...

[the remark this answers is not in the document]
Comment (ludmilamarian) [reply cds_dojson/marc21/fields/utils.py]: If I remember correctly we did update only the authors that had CDS Ids, but that does not mean that only that ID is `non curated`.  It's the entire list of IDs that is linked to one author. If the program made a wrong guess, then all the IDs are going to be wrong.

File: tests/test_books.py [hunk 3]
```diff
@@ -46,6 +46,8 @@ def check_transformation(marcxml_body, json_body):
         }
     }
     expected.update(**json_body)
+    print(record, '======================')
+    print(expected, '+++++++++++++++++++++')
```
Comment (ludmilamarian) [inline tests/test_books.py]: leftover?

**Pull request merged**